lte/gateway/python/magma/pipelined/tests/test_uplink_bridge.py

```diff
@@ -505,7 +505,7 @@ def testFlowSnapshotMatch(self):
         self.assertIn(cls.SGi_GW, get_iface_gw_ipv4(cls.UPLINK_BRIDGE),
                       "gw not found")
 
-
+@unittest.skip
```

Comment:
Why is it off?

Reply:
This changes resolve conf on the AGW, this is already skipped. I just moved skip to class to avoid test setup.